src/VirtoCommerce.CatalogModule.Data/Services/CategoryMover.cs

```diff
@@ -51,5 +56,21 @@ public override async Task<List<Category>> PrepareMoveAsync(ListEntriesMoveReque
 
             return result;
         }
+
+        protected virtual async Task ValidateOperationArguments(ListEntriesMoveRequest moveInfo)
+        {
+            ValidateParameters(moveInfo);
+
+            var validator = new ListEntriesMoveRequestValidator(_categoryService);
+            await validator.ValidateAndThrowAsync(moveInfo);
+        }
+
+        private void ValidateParameters(ListEntriesMoveRequest moveInfo)
```

Comment:
I think is better to disable this stupid validation rule

Reply:
Done